lpc55_areas/src/lib.rs

```diff
@@ -31,9 +46,9 @@ impl BootField {
         BootField {
             // 0 = TZ-M enabled image. Images are only built
```

Comment:
nit: this comment about it being zero is now less useful and possibly distracting (same below)

Reply:
Yup, deleted those comments now that they're strongly typed